websites/U/UKTV/presence.ts

```diff
@@ -0,0 +1,111 @@
+const presence = new Presence({
+	clientId: "1002899869599551508",
+});
+
+async function getStrings() {
+	return presence.getStrings(
+		{
+			play: "presence.playback.playing",
+			pause: "presence.playback.paused",
+			browse: "presence.activity.browsing",
```

Comment:
`presence.` strings will be removed soon, so using `general.` is btter
```suggestion
			play: "general.playing",
			pause: "general.paused",
			browse: "general.browsing",
```